src/detect-smb-version.c

```diff
@@ -0,0 +1,159 @@
+/* Copyright (C) 2022-2023 Open Information Security Foundation
```

Comment:
Very sorry for not noticing this before, but technically speaking, since we're only adding this now, this should be 2023 only, shouldn't it?

Reply:
With the original commits/work from Eloy being from last year, I just added this year. Of course willing to update with whatever you all see fit.